src/System.Runtime.InteropServices.RuntimeInformation/tests/DescriptionNameTests.cs

```diff
@@ -44,31 +44,17 @@ public void DumpRuntimeInformationToConsole()
 
         [Fact]
         [SkipOnTargetFramework(~TargetFrameworkMonikers.Netcoreapp)]
-        public void VerifyRuntimeDebugNameOnNetCoreApp()
+        public void VerifyRuntimeNameOnNetCoreApp()
         {
-            AssemblyFileVersionAttribute attr = (AssemblyFileVersionAttribute)(typeof(object).GetTypeInfo().Assembly.GetCustomAttribute(typeof(AssemblyFileVersionAttribute)));
-            string expected = string.Format(".NET Core {0}", attr.Version);
-            Assert.Equal(expected, RuntimeInformation.FrameworkDescription);
+            Assert.True(RuntimeInformation.FrameworkDescription.StartsWith(".NET Core"));
```

Comment:
Nit: could you add FraneworkDescription as part of a second argument so we're see what it is if the assert fails?